apiserver/facades/client/action/action_test.go

```diff
@@ -243,17 +243,17 @@ func (s *actionSuite) TestEnqueue(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(res.Results, gc.HasLen, 4)
 
-	expectedError := &params.Error{Message: "id not found", Code: "not found"}
 	emptyActionTag := names.ActionTag{}
-	c.Assert(res.Results[0].Error, gc.DeepEquals, expectedError)
+	c.Assert(res.Results[0].Error, gc.DeepEquals, &params.Error{Message: fmt.Sprintf("%s not valid", arg.Actions[0].Receiver), Code: ""})
```

Comment:
If the CodeNotValid is added, the code can continue to do what the original code did and compare the errors to a params.Error{} with the correct Code set